nautobot/extras/api/views.py

```diff
@@ -484,6 +484,7 @@ class ImageAttachmentViewSet(ModelViewSet):
     queryset = ImageAttachment.objects.all()
     serializer_class = serializers.ImageAttachmentSerializer
     filterset_class = filters.ImageAttachmentFilterSet
+    parser_classes = [JSONParser, MultiPartParser]
```

Comment:
I don't think we support CSV export of image attachments, but including `NautobotCSVParser` here might be a good idea regardless.

Reply:
Done!

Reply:
Actually, looks like that broke CI?

Reply:
No, that failure's an intermittent one that we haven't been able to root-cause yet. I just kicked off a rerun which will probably pass.